ui/components/app/confirmation/confirm-footer/confirm-footer.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import classnames from 'classnames';
+import {
+  Box,
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '../../../component-library';
+import {
+  BlockSize,
+  Display,
+  FlexDirection,
+} from '../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import { BoxProps } from '../../../component-library/box';
+import { ConfirmFooterProps } from './confirm-footer.types';
+
+const ConfirmFooter: React.FC<ConfirmFooterProps> = ({
+  className = '',
+  cancelText,
+  confirmText,
+  onCancel,
+  onConfirm,
+  confirmButtonProps,
+  cancelButtonProps,
+  ...props
+}) => {
```

Comment:
None of the props should be required, the component is used at only 1 place and will get required info using hook.

Please refer: https://github.com/MetaMask/metamask-extension/pull/22081/files#diff-81455140e7e7ab4c7ec7834577a0b4f81e41361fabaf15737d8f67530ed6af07R12